src/dialogs/auth/definition.ts

```diff
@@ -1,11 +1,40 @@
 import { Struct } from 'src/structs';
 
 export type Auth =
-  | Struct<'logoutDialog', {
-    success?: VoidCallback;
-    error?: ErrorCallback;
-  }>
-  | Struct<'confirmLockDialog', {
-    success?: VoidCallback;
-    error?: ErrorCallback;
-  }>
+  | Struct<
+      'logoutDialog',
+      {
+        success?: VoidCallback;
+        error?: ErrorCallback;
+      }
+    >
+  | Struct<
+      'confirmLockDialog',
+      {
+        success?: VoidCallback;
+        error?: ErrorCallback;
+      }
+    >;
+
+export type AccountApplication =
+  | Struct<
+      'adminDialog',
```

Comment:
Name is too generic which leads to various meaning.
```suggestion
      'adminApplicationDialog',
```

Reply:
Generic name can mis-understood by other developers as createAdminDialog, editAdminDialog, deleteAdminDialog and so on... It's best practice to include action verb in your dialog name explaing the operation

Reply:
I'm aware sir. I was planning to change it to non-generic name since at that time, I couldn't think of an appropriate name for it. I just forgot to do that. Thanks for notifying me sir.